student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, age, classroom, name: 'Unknown', parent_permission: true)
```

Comment:
:orange_book: [OPTIONAL] The id is automatically generated by the parent class, so there is no need to include it as a constructor parameter.